mocks/python_bindings/Makefile

```diff
@@ -63,16 +63,16 @@ $(VPF_mocks_DIR)/lib$(VPF_mocks_LIB).a: $(VPF_mocks_DIR)/*.c $(VPF_mocks_DIR)/*.
 	$(MAKE) -C $(VPF_mocks_DIR) libs
 
 sharedlib: $(PROJECT).so
-ifeq ($(FIX_PYTHON_LINK), 1)
-	@{ \
-		CURRENT_PYTHON_LIB=`otool -L $(PYTHON_EXTN) | grep -i python | cut -d " " -f1 | grep -v $(PROJECT) | xargs` ; \
-		PYTHON_LIB_NAME=$(PYTHON_LIB_BASE).dylib ; \
-		LINK_PYTHON_LIB=$(PYTHON_LIBDIR)/$$PYTHON_LIB_NAME ;\
-		if [[ "$$CURRENT_PYTHON_LIB" != "$$LINK_PYTHON_LIB" ]] ; then \
-		install_name_tool -change $$CURRENT_PYTHON_LIB $$LINK_PYTHON_LIB $(PYTHON_EXTN); \
-		fi ;\
-	}
-endif
+# ifeq ($(FIX_PYTHON_LINK), 1)
+# 	@{ \
+# 		CURRENT_PYTHON_LIB=`otool -L $(PYTHON_EXTN) | grep -i python | cut -d " " -f1 | grep -v $(PROJECT) | xargs` ; \
+# 		PYTHON_LIB_NAME=$(PYTHON_LIB_BASE).dylib ; \
+# 		LINK_PYTHON_LIB=$(PYTHON_LIBDIR)/$$PYTHON_LIB_NAME ;\
+# 		if [[ "$$CURRENT_PYTHON_LIB" != "$$LINK_PYTHON_LIB" ]] ; then \
+# 		install_name_tool -change $$CURRENT_PYTHON_LIB $$LINK_PYTHON_LIB $(PYTHON_EXTN); \
+# 		fi ;\
+# 	}
+# endif
```

Comment:
Ready to be removed, or do you still want to keep this around?

Reply:
Might as well remove them